.nuget/packages.config

```diff
@@ -0,0 +1,5 @@
+﻿<?xml version="1.0" encoding="utf-8"?>
+<packages>
+  <package id="RavenDB.Server" version="2.0.2360" />
```

Comment:
Probably inherited, yeah. I think it was required for the global test suite, which we're not using, I'll go through those later and trim the config file down to required references.